classes/village/VillageManager.php

```diff
@@ -1589,15 +1625,6 @@ public static function enactProposal(System $system, User $player, int $proposal
                 $message = "Accepted peace offer from " . VillageManager::VILLAGE_NAMES[$proposal['target_village_id']] . "!";
                 // create notification
                 self::createProposalNotification($system, $proposal['village_id'], NotificationManager::NOTIFICATION_PROPOSAL_PASSED, $proposal['name']);
-                // if village occupied by enemy, return to owner's control
-                $system->db->query("UPDATE `region_locations`
-                    INNER JOIN `regions` on `regions`.`region_id` = `region_locations`.`region_id`
-                    SET `region_locations`.`occupying_village_id` = NULL
-                    WHERE `regions`.`village` = {$proposal['target_village_id']} AND `region_locations`.`occupying_village_id` = {$proposal['village_id']}");
-                $system->db->query("UPDATE `region_locations`
-                    INNER JOIN `regions` on `regions`.`region_id` = `region_locations`.`region_id`
-                    SET `region_locations`.`occupying_village_id` = NULL
-                    WHERE `regions`.`village` = {$proposal['village_id']} AND `region_locations`.`occupying_village_id` = {$proposal['target_village_id']}");
```

Comment:
What's the motivation for removing this behavior?

Reply:
I don't think we strictly want to remove this functionality just yet, but its part of decoupling town control from region control. So even if you trade a region you control to someone else it doesn't steal town control from someone else.